src/resolvers/EvolveResolver.ts

```diff
@@ -0,0 +1,20 @@
+
+import { Resolver, Query, Mutation, Arg } from "type-graphql";
+import { EvolvingService } from "../services/EvolvingService";
+import { EvolveInput } from "../types/graphql/inputs/EvolveInput";
+import { EvolveResponse } from "../types/graphql/outputs/EvolveOutput";
+
+@Resolver()
+export class EvolveResolver {
+  constructor(private evolvingService: EvolvingService) {}
+
+  @Query(() => String)
+  evolveStatus() {
+    return "Evolution in progress!";
```

Comment:
change message to something like "use mutation"

Reply:
removed